packages/template-retail-react-app/package.json

```diff
@@ -83,7 +83,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "270 kB"
+            "maxSize": "280 kB"
```

Comment:
Is this still something that was required? Is a full increment of 10kb something we need.. or can it be 5.. or 2 or 1?

Reply:
Good news, I re-run the file-size check, and the size does not exceed 270kb anymore. The exceeded size probably came from the react-query work that this PR was based on before.